Dockerfile-centos7

```diff
@@ -35,11 +36,11 @@ RUN cd /opt && wget https://github.com/appneta/tcpreplay/releases/download/v${TC
 # Install Python 3.8
 RUN cd /opt && wget https://www.python.org/ftp/python/${PYTHON_VERSION}/Python-${PYTHON_VERSION}.tgz \
   && tar xzf Python-${PYTHON_VERSION}.tgz \
-  && cd /opt/Python-${PYTHON_VERSION} && ./configure --enable-optimizations \
```

Comment:
Not that I think it matters much, but just curious - why removing the `--enable-optimizations` flag?

Reply:
It doesn't build with it.

Google the issue and people complains about this option. I didn't investigate further